### PR TITLE
Issue #738 replace .inc configuration files by .ttl (turtle) files.

### PR DESCRIPTION
**Pull request by kinow**
The cache that was created in the `Model` instance, now was moved to the
`GlobalConfig`, but is still accessed from `Model` to add entries looked up.

There is no more need of a separate `.ttl` file for vocabularies, as it is
now part of the configuration file. Old files were removed.

In order to maintain backward compatibility, most methods of `GlobalConfig`
and of `VocabularyConfig` were maintained as-is. With internal changes to
re-route requests to the right RDF objects.

Tests were minor updates in tests, but only to replace `.inc` by `.ttl` or fix
coverage annotations.

## Tests

Built locally, all tests passed. Built the master branch later, with same test fuseki (initialized via the Shell script in the `tests` folder). No difference in the output.

```
OK, but incomplete, skipped, or risky tests!
Tests: 346, Assertions: 503, Skipped: 2, Risky: 8.
```

Tested running with `php -S 0.0.0.0:8000`, keeping an eye on logs and at the UI to see if there was any issue, everything looked OK except I didn't have the language library, so the language dropdown did not work (same happens for current version from master branch).

Tested from Docker as well. Found a minor issue in the `Dockerfile` (sending separate pull request for that), but everything else worked, after I followed the instructions from the Wiki.

screenshot from 2018-06-02 17-36-17

If this pull request gets approved/merged, we would still have to update the existing configuration for the configuration & installation of Skosmos, as well as the Docker documentation that mentions configuration files as well.

### CODE REVIEW

**Comment by osma** [comment]
Fantastic work @kinow!

In addition to documentation updates I think we could also use a conversion tool that reads existing config.inc and outputs corresponding Turtle. But we can handle that in a separate issue.

I think skosmos:languages should be an RDF list (plain parentheses in Turtle) as discussed on the issue.

Will do more detailed review early next week!

**Comment by kinow** [comment]
I believe my RDF-fu is failing me :sob: . Maybe you could provide an example and I can adjust the code after that?

And a detailed review will be much appreciated! I wasn't familiar with the `Model` and its cache, but already learned a few more things about Skosmos` code base.

Thanks @osma !

File: model/Model.php [hunk 1]
```diff
@@ -423,7 +344,7 @@ private function createDataObjects($class, $resarr)
     public function getVocabularies()
```
[the remark this answers is not in the document]
Comment (kinow) [reply model/Model.php]: I agree. Just kept them just to reduce the size of the changes. But happy to move everything to `GlobalConfiguration` if you think that'd be better that way.

**Comment by osma** [comment]
After changing to collections in the Turtle file, the object of the `skosmos:languages` property should become an EasyRdf Collection object, which can be iterated with `foreach` as if it were a PHP array. Here is an example: https://github.com/njh/easyrdf/blob/master/test/EasyRdf/CollectionTest.php#L97

File: model/Model.php [hunk 2]
```diff
@@ -19,8 +9,6 @@ class Model
 {
     /** EasyRdf\Graph graph instance */
     private $graph;
```
[the remark this answers is not in the document]
Comment (kinow) [reply model/Model.php]: Yup! Good catch!

[the remark this answers is not in the document]
Comment (osma) [reply model/Model.php]: Thank Scrutinizer for that... (I have the browser plugin installed and it flags issues like this inline when you look at the GitHub diffs)

**Comment by kinow** [comment]
Thanks for the lists example @osma ! My first try was with `( )`, but I was doing

```
skosmos:languages (
  [ :name "en" ; :value "English" ] ,
  [... ]
) ;
```

But due to that extra comma the syntax was invalid. I read about RDF collections (bit confusing with those `first`, `last`, etc... and then found a way that worked in Turtle.

Already implemented the change locally, following the examples you provided. Now running tests before I push it :tada:

**Comment by osma** [comment]
Thanks a lot @kinow !

**Comment by kinow** [comment]
Thank you for testing and for the help with the languages list. Learned a lot here ! :D

**Pull request merged**